packages/unlock-js/jest.config.js

```diff
@@ -9,6 +9,6 @@ module.exports = {
     '<rootDir>/src/__tests__/setup/',
   ],
   transform: {
-    '^.+\\.js$': 'babel-jest',
+    '^.+\\.jsx?$': require.resolve('./jest.transform.js'),
```

Comment:
On a du jsx ici?

Reply:
nah I think the question mark after the x in the regex makes it optional. Changed to js anyway